algos/eq/equalizers_tuning.rst

```diff
@@ -0,0 +1,924 @@
+
+.. code-block:: bash
+
+		ssh user@aa.bb.cc.dd "aplay -l"
+
+Since the tests are done with low-level ALSA aplay and arecord
+utilities it is recommended to temporarily rename in DUT the audio
+servers to disable them. Kill manually the processes or reboot to
+avoid them continue running. The audio servers can be disabled from OS
+system control in more elegant way but it is harder to remember how to
+do it and restore to normal vs. the brute force way.
+
+.. code-block:: bash
+
+		cd /usr/bin
+		sudo mv pulseaudio pulseaudio.disabled
+		sudo mv pipewire pipewire.disabled
+
+
+Frequency response measurement
+******************************
+
+Note: More professional audio analyzer systems are recommended to be
+used for final tuning. The procedures described in this document are
+for coarse initial settings. Final tuning, especially if dependence
+to regulations and standards need to be done with care in professional
+environment with calibrated measurement equipment.
+
+To measure speakers an omnidirectional USB measurement microphone is
+recommended, e.g. UMM6 [2]_ or UMIK-1 [3]_. Such microphones are
+inexpensive and do not necessarily have a flat frequency response but
+the manufacturers provide a serial number based downloadable
+calibration file for them. The calibration can be applied to these
+measurements in SOF as well by referencing the downloaded calibration
+data to measurement script.
+
+Next step up are analog condenser measurement microphones with a
+high-end USB sound card that can provide the 48V phantom voltage. But
+analog microphones add more calibration consideration for analog
+level. The measurement microphones can be also calibrated for absolute
+level with dedicated microphone calibrators those can output into the
+sealed compartment a reference 94 dBSPL tone.
+
+The tools for measurement and EQ design are in located in directory
+$SOF_WORKSPACE/sof/tools/tune/eq. The test setup is such that the DUT
+device plays back the measurement wav file via ssh commands and the
+development PC connected USB microphone captures the output. To
+achieve this the configuration files for playback and capture need to
+be edited.
+
+The capture device UMM6 is hw:3.0 (card 3, device 0), this can be seen
+from output of arecord command on a the development PC example. We
+also know that this device supports one capture channel.
+
+.. code-block:: bash
+
+		arecord -l
+		**** List of CAPTURE Hardware Devices ****
+		card 0: PCH [HDA Intel PCH], device 0: ALC257 Analog [ALC257 Analog]
+		  Subdevices: 1/1
+		  Subdevice #0: subdevice #0
+		card 1: Ultra [Fast Track Ultra], device 0: USB Audio [USB Audio]
+		  Subdevices: 1/1
+		  Subdevice #0: subdevice #0
+		card 2: Audio [ThinkPad Dock USB Audio], device 0: USB Audio [USB Audio]
+		  Subdevices: 1/1
+		  Subdevice #0: subdevice #0
+		card 3: UMM6 [UMM-6], device 0: USB Audio [USB Audio]
+		  Subdevices: 1/1
+		  Subdevice #0: subdevice #0
+
+The settings file
+
+.. code-block:: bash
```

Comment:
When the "bash" is used as the language mode for styling while it is just 'cat' followed by a content of a file that uses different language, some strange color styling appears. Might be better to just paste the content and put the file name next it it.

Reply:
I wonder if there's a suitable code-block style for generic ASCII stuff. Do you have a recommendation? Just pasting the content without code-block would apply text formatting to it that would mess it up.

Reply:
You may replace it with (note the `::` at the end of the previous paragraph)
```
The settings file::

   The content of the file (each line indented)
   that is not processed and rendered using monospace font.
```